### PR TITLE
Removed usage of deprecated methods, moved symfony extensions from component to bundle

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

**Comment by chives** [comment]
@fsi-open

**Comment by rn0** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

File: DataGrid/Extension/Symfony/ColumnTypeExtension/FormExtension.php [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+                        $field['options']['input'] = 'datetime';
+                    }
+                }
+                break;
+        }
+
+        $formBuilderOptions = array(
+            'type' => new RowType($fields),
+            'csrf_protection' => false,
+
```
Comment (szymach) [inline DataGrid/Extension/Symfony/ColumnTypeExtension/FormExtension.php]: empty line

File: Resources/docs/extensions/symfony.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+## Event Subscribers provided by extension ##
+
+``FSi\Bundle\DataGridBundle\DataGrid\Extension\Symfony\EventSubscriber\BindRequest``
+
+## Column Type Extensions provided by extension ##
+
+``FSi\Bundle\DataGridBundle\DataGrid\Extension\Symfony\ColumnTypeExtension\FormExtension``
+
+This extensions is loaded into almost all column types. It allows you to set 
+``editable`` otpion in column type.
```
Comment (szymach) [inline Resources/docs/extensions/symfony.md]: *set the ``editable`` option in column type.

File: Tests/DataGrid/Extension/Symfony/ColumnType/ActionTest.php [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+               )
+           ),
+           $this->column->filterValue(array(
+               'foo' => 'bar'
+           ))
+       );
+    }
+}
+
+class MyRequest extends Request
```
Comment (szymach) [inline Tests/DataGrid/Extension/Symfony/ColumnType/ActionTest.php]: aren't two classes in one file a bad practice?

File: Tests/DataGrid/Extension/Symfony/ColumnType/ActionTest.php [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+                'absolute' => true
+            )
+        ));
+
+       $this->assertSame(
+           array(
+               'edit' => array(
+                   'content' => 'edit',
+                   'field_mapping_values' => array(
+                           'foo' => 'bar'
```
Comment (szymach) [inline Tests/DataGrid/Extension/Symfony/ColumnType/ActionTest.php]: double tab

**Comment by szymach** [comment]
aside from the few small things mentioned above :+1:

**Pull request merged**